tests/fixtures/env-wrong/bad-quotes.env

```diff
@@ -0,0 +1 @@
+FOO=\"\\\\\\\\\n
```

Comment:
@danielstokes Could you come up with a test case that causes a preg error please, so we can merge this?